### PR TITLE
Sign up with Xero: new page

### PR DESCRIPTION
**Pull request by pzaichkina**
# Description

New page added to cover the Sign Up with Xero content from Zendesk in the public docs. 
docs/integrations/accounting/xero/xero-sign-up-flow

## Type of change

- [X] New document(s)/updating existing

### CODE REVIEW

File: docs/integrations/accounting/xero/xero-sign-up-flow.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+---
+title: "Build a Sign Up with Xero flow"
+description: "Learn how to enable the Sign Up with Xero flow with Codat to support your app's certification"
+---
+
+Codat has made it easier for you to implement **Sign Up with Xero**. It is a flow that allows you to quickly onboard your potential customers using pre-populated data from their Xero accounting software and show them the unique value of your app faster. 
+
+## When to build?
```
Comment (mcclowes) [inline docs/integrations/accounting/xero/xero-sign-up-flow.md:8]: I really like this header convention 👍

File: docs/integrations/accounting/xero/xero-sign-up-flow.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+When this link is initiated by the customer, it triggers the creation of a company and a connection in Codat, and redirects them to Xero login for authorization. Once authorized, they will be redirected to your **Sign up success redirect URL**.
+
+### Set the Sign up success redirect URL
+
+Navigate to **Settings > Auth flow > Redirects** to access the [Redirects](https://app.codat.io/settings/redirects) page in the Codat Portal. Under **Sign up success redirect URL**, enter the base URL of the form you will display to the customer, without any parameter replacements. Your changes will be auto-saved.
+
+![](/img/integrations/accounting/xero/0021-xero-success-redirect-url.png)
+
+### Utilize Xero data
```
Comment (mcclowes) [inline docs/integrations/accounting/xero/xero-sign-up-flow.md:63]: Can we be more specific for this header?

File: docs/integrations/accounting/xero/xero-sign-up-flow.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+### Retrieve the Sign Up with Xero URL
+
+**Sign Up with Xero URL** is the URL that will be shared with the end user for them to authorise access to Xero. It is generated by Codat, but requires you to action the following: 
+
+1. Retrieve `clientId` and `platfromKey`
+
+   These two values will form part of the **Sign Up with Xero URL**. 
+
+   Call our `GET https://api.codat.io/profile/syncSettings` endpoint and pick up the `clientId` returned in the response. For Xero, the `platfromKey` value is `gbol`, as captured in our [accounting platform key list](https://docs.codat.io/integrations/accounting/accounting-platform-keys).
```
[the remark this answers is not in the document]
Comment (pzaichkina) [reply docs/integrations/accounting/xero/xero-sign-up-flow.md:48]: @mcclowes this is the endpoint that is not in our API that I mentioned in slack

File: docs/integrations/accounting/xero/xero-sign-up-flow.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+1. Retrieve `clientId` and `platfromKey`
+
+   These two values will form part of the **Sign Up with Xero URL**. 
+
+   Call our `GET https://api.codat.io/profile/syncSettings` endpoint and pick up the `clientId` returned in the response. For Xero, the `platfromKey` value is `gbol`, as captured in our [accounting platform key list](https://docs.codat.io/integrations/accounting/accounting-platform-keys). 
+   
+2. Generate the URL
+
+   Use the values retrieved previously to generate the URL with the following structure: `https://link-api.codat.io/clients/{ClientID}/signUpWith/{PlatformKey}`  
+   Note that this URL should also be used as a response to _6. Purchase/sign up page URL_ in your App Store listing.
```
Comment (d-laing) [inline docs/integrations/accounting/xero/xero-sign-up-flow.md:53]:
```suggestion
   Note that this URL should also be used as a response to checkpoint _6. Purchase/sign up page URL_ in your App Store listing. 
```

**Review by d-laing — CHANGES_REQUESTED** [review]
Looks good. I made a few comments.

**Review by d-laing — APPROVED** [review]
The additions to the intro look good.

**Pull request merged**